packages/react-scripts/template/src/components/example/ArtifactsViewer.js

```diff
@@ -30,13 +30,26 @@ const ArtifactsViewSkeleton = () => {
 }
 
 const ArtifactsViewer = ({ user: { cisId } }) => {
-  // Use our custom hook
-  const [{ loading, artifacts, photos, error }] = useArtifacts(cisId)
+  const artifactsUrl = `/service/memories/presentation/patrons/${cisId}/persons?numTaggedPersonArtifacts=3&includeTaggedPersons=true&includeEmptyPersons=false&includeHistory=false`
```

Comment:
I typically prefer to do query params as a map/object because I find it easier to read and maintain, but you are just moving existing code, so nothing we need to fix. But I wonder if we should consider having a recommendation or pattern. I'll add a card to FAWG to discuss.

Reply:
Yes, let's discuss.